ClassRegistration/ClassRegistration.App/Controllers/SectionController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ClassRegistration.DataAccess.Entity;
+using ClassRegistration.Domain.Repositories;
+using ClassRegistration.Domain.Interfaces;
+
+
+
+namespace ClassRegistration.App.Controllers
+{
+
+    [Route("api/[controller]")]
+    public class SectionController : ControllerBase
+    {
+
+        private readonly ISectionRepository _secRepo;
+        
+
+        public SectionController(ISectionRepository secRepo)
+        {
+            _secRepo = secRepo;
+           
+        }
+
+
+
+        // GET api/section/items/50
+        [HttpGet("items/{id}")]
+        public ActionResult<Section> GetCourseByInstructorID(int id)
```

Comment:
It may seem strange, but in this case you'd want to pull the id from the body. The reason is because the way these methods are constructed effects their URL, and `items/id` indicates that we're getting a set of items from a course of a particular id. Instead, the pattern that would make more sense here is just to leave the Get URL as it's default (`[HttpGet]`), and then have the method take in an integer named `instructorId`. This will result in a final request that looks like: `GET /api/section?instructorId=x`

Reply:
Yes, I agree that makes sense, will push a new commit with this change.